runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -156,6 +159,25 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (config.callChangeAudioCue() && currentRound != null)
```

Comment:
Performing a null check on `currentRound` is cheaper than a configuration check, so flip the order of these.
```suggestion
		if (currentRound != null && config.callChangeAudioCue())
```